### PR TITLE
Updates cloud-kms samples to use v1 libraries

### PR DESCRIPTION
**Pull request by philcoakley**

### CODE REVIEW

**Comment by philcoakley** [comment]
@broady Are you able to take a look at this?

File: kms/crypter/crypter.go [hunk 1]
```diff
@@ -14,7 +14,8 @@ import (
 
 	"golang.org/x/net/context"
 	"golang.org/x/oauth2/google"
-	cloudkms "google.golang.org/api/cloudkms/v1beta1"
+q
```
Comment (broady) [inline kms/crypter/crypter.go]: spurious character

**Comment by broady** [comment]
LGTM except for the typo.

**Comment by broady** [comment]
thanks!

**Comment by philcoakley** [comment]
Thank you!

**Pull request merged**